<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Codys-Mac.local" ignored="0" name="Command line test db446eda-3129-4fda-8023-dfd8f3081ad4" tests="2" failures="2" timestamp="23 Mar 2018 07:03:25 GMT" time="891.127" errors="0">
  <testcase name="prankcaller" time="230.222" classname="test.tests.launchApp.Tests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test.tests.launchApp.Tests.prankcaller(Tests.java:167)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- prankcaller -->
  <testcase name="launchApp" time="274.918" classname="test.tests.launchApp.Tests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test.tests.launchApp.Tests.launchApp(Tests.java:355)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- launchApp -->
</testsuite> <!-- Command line test db446eda-3129-4fda-8023-dfd8f3081ad4 -->
